internal/cloud/realm/error.go

```diff
@@ -3,13 +3,31 @@ package realm
 import (
 	"bytes"
 	"encoding/json"
+	"errors"
 	"net/http"
 )
 
 const (
 	invalidSessionCode = "InvalidSession"
 )
 
+// ErrInvalidSession is an invalid session error that has a follow up message
+type ErrInvalidSession struct {
+	error
+}
+
+func (err ErrInvalidSession) SuggestedCommands() []string {
+	return []string{}
+}
+
+func (err ErrInvalidSession) ReferenceLinks() []string {
```

Comment:
you can remove this function declaration, `ErrInvalidSession` will not need to provide any ReferenceLinks at this time